packages/peregrine/lib/store/reducers/cart.js

```diff
@@ -43,7 +46,6 @@ const reducerMap = {
             return {
                 ...state,
                 detailsError: payload,
-                cartId: null,
```

Comment:
Removing `cartId` should be done through `removeCart`.

Reply:
This makes me nervous why this existed, but I couldn't find any components observing this state, so seems safe 🤞